### PR TITLE
CVE-2019-10149

### PR DESCRIPTION
**Pull request by rehakm**

### CODE REVIEW

File: 2019/10xxx/CVE-2019-10149.json [hunk 1]
```diff
@@ -4,15 +4,67 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2019/10xxx/CVE-2019-10149.json]: 'vendor_name' is a required property

**Comment by iamleot** [comment]
Hello rehakm, 
can you please add as reference also upstream advisory:

 <https://www.exim.org/static/doc/security/CVE-2019-10149.txt>

(unfortunately at the moment it seems there are not a lot of details.)


Thank you!

**Comment by rehakm** [comment]
Added upstream advisory as requested.

File: 2019/10xxx/CVE-2019-10149.json [hunk 2]
```diff
@@ -4,15 +4,70 @@
...
```
Comment (cve-team) [inline 2019/10xxx/CVE-2019-10149.json]: Invalid JSON file

File: 2019/10xxx/CVE-2019-10149.json [hunk 3]
```diff
@@ -4,15 +4,71 @@
...
```
Comment (cve-team) [inline 2019/10xxx/CVE-2019-10149.json]: Invalid JSON file

**Pull request merged**